mlflow/server/js/src/shared/web-shared/design-system/index.tsx

```diff
@@ -7,15 +7,15 @@ export type DarkModePref = 'system' | 'dark' | 'light';
 const DARK_MODE_PREF_DEFAULT = 'light';
 
 export interface SupportsDuBoisThemesProps {
-  disabled?: boolean;
+  isDarkMode?: boolean;
```

Comment:
do we have `isDarkMode` in universe?

Reply:
i'm actually not too sure how the Databricks MLflow UI picks up the dubois theme. @hubertzub-db do you know?

Reply:
so in universe, `isDarkMode` is a prop of `DesignSystemThemeProvider`, not `<SupportsDuBoisThemes>`:
https://src.dev.databricks.com/databricks/universe/-/blob/design-system/src/design-system/DesignSystemProvider/DesignSystemProvider.tsx?L12

It seems that it should be perfectly fine to use `DesignSystemThemeProvider` directly (with supported prop) and ditch `SupportsDuBoisThemesProps`

Reply:
> Option 2 is already implemented in this PR, did you mean to implement it in databricks as well?

Actually the opposite - the way it's implemented right now (expanded props of `SupportDuBoisThemes` component) makes it difficult to import it back to Databricks. 
I believe we can fit 100% of the new logic inside `SupportDuBoisThemes` stub (e.g. somewhere [here](https://github.com/mlflow/mlflow/pull/10169/files#diff-321d4aeff014accfd371d1fa8990b5b4dd17ba48425484f532bab873cb522ff7R17)) and expose a function for changing the state e.g. via context so it can be changed downstream (in the header). I think this should be easiest way to solve it.

> Are there any other blockers for this PR being merged, Hubert?

AFAICT the setting is not persisted yet - shouldn't it be included in this changeset?

https://github.com/mlflow/mlflow/assets/104438646/dd54204c-be5e-41a4-bc51-f6c748ac530d